### DISCUSSION TITLE
Multistore image problem

### DISCUSSION THREAD

**Discussion by LaqueP** [PrestaShop/PrestaShop#33546] [Q&A] [answered]
Prestashop 8.0.4

We have two stores on the same prestashop, one for B2C and one for marketplaces.

We require different images to be managed in each store. However, when we upload and select the images for each combination, one store breaks the combinations of the other shop.

Has anyone had this problem before and solved it?

**Comment by MatShir** [comment, marked as answer]
I think this feature is fixed on 8.1 new product page. I sugget to do some test on 8.1 and if it works to plan an upgrade. If the issue is still persistent on 8.1, I will transfer this discussion into a bug.
